addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -224,6 +240,66 @@ $o-btn-outline-border-width-defaults: () !default;
 $o-navbar-nav-pills-link-padding-x: if(o-website-value('header-links-style') == 'pills', 1rem * 1.5, null) !default;
 $o-navbar-nav-pills-link-border-radius: if(o-website-value('header-links-style') == 'pills', 10rem, null) !default;
 
+// Borders
+$border-radius-sm: .3rem !default;
+$border-radius: .4rem !default;
+$border-radius-lg: .6rem !default;
+$border-radius-xl: 1rem !default;
+$border-radius-pill: 50rem !default; // BS Default
+
 // Accordion
 $accordion-color: inherit !default;
 $accordion-bg: inherit !default;
+$accordion-padding-x: 1.5rem !default;
+$accordion-border-width: 1px !default;
+$accordion-border-color: $border-color !default;
+$accordion-button-active-bg: $light !default;
+$accordion-button-active-color: color-contrast($light) !default;
+$accordion-button-focus-box-shadow: none !default;
+$accordion-icon-color: $body-color !default;
+$accordion-icon-active-color: $accordion-button-active-color !default;
+
+// Badge
+$badge-padding-y: .3em !default; // Ensure circle(-ish) badges for 1 digit numbers
+$badge-padding-x: .6em !default; // " "
+$badge-border-radius: $border-radius-pill !default;
+
+//Popover
+$popover-bg: $body-bg !default;
+$popover-border-color: $border-color !default;
+$popover-arrow-color: $body-bg !default;
+$popover-arrow-outer-color: $border-color !default;
+
+// Shadows
+$box-shadow: 0px 4px 16px rgba($black, 0.12) !default;
+$box-shadow-sm: 0px 1px 3px rgba($black, .1) !default;
+$box-shadow-lg: 0px 12px 32px rgba($black, .175) !default;
+
+// Cards
+$card-border-color: $border-color !default;
+
+// Modals
+$modal-header-border-width: 0 !default;
+$modal-footer-border-width: 0 !default;
+
+// Alert
+$alert-border-scale: -30% !default;
+
+// Close
+$btn-close-color: $body-color !default;
```

Comment:
I see why (ensure that the button matches the use base color?), but I'm not sure if its worth 🤔...

At the end what matters is that the contrast provided against the component bg (eg modal), and we don't enforce customizations for these elements and, for these ones,  we neither provide options through the "Theme" tab.
 
I'd keep the BS default (until we'll provide customization for the components where this buttons is rendered)